pyriemann/datasets/simulated.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+from sklearn.utils.validation import check_random_state
+from .sampling import generate_random_spd_matrix, sample_gaussian_spd
+
+
+def make_gaussian_blobs(n_samples=100, n_dim=2, class_sep=1.0, class_disp=1.0,
+                        random_state=None):
+    """Generate SPD dataset with two classes sampled from Riemannian Gaussian
```

Comment:
Do you think that it is possible to generalize this function two `k` classes?

Reply:
If you look at `sklearn.datasets.make_classification` they have the option of creating several classes, but the centroids are put into vertices of a hypercube. This means that the pairwise distance between the class means is not the same. 

I would like to know your opinion (@agramfort @sylvchev @qbarthelemy) on whether we should do it like this or if you are aware of more sophisticated ways of generating the coordinates for `n_classes` centroids in `n_dim` dimensions, while making sure that their pairwise distance is controlled by `class_sep`.

Reply:
You are right: vertices of an hypercube are not equidistant.
@Marco-Congedo, do you know a method to generate `n` equidistant SPD matrices?

But, ok for me to merge the function as is, and to open an issue on this subject to generalize the function later for `n>2`.

Reply:
I think it is possible to find a sophisticated way for generalizing this to more than two classes, but I don't have any simple way of doing this at hand. I agree with Quentin: I'll prefer to leave the code like this, with 2-class, and open an issue.